.gitmodules

```diff
@@ -58,3 +58,9 @@
 [submodule "auxil/out_ptr"]
 	path = auxil/out_ptr
 	url = https://github.com/soasis/out_ptr.git
+[submodule "auxil/spicy-plugin"]
+	path = auxil/spicy-plugin
+	url = https://github.com/zeek/spicy-plugin
```

Comment:
Have the commits used for these been given a tag? I think we said we were going to try to pin to specific release tags, right?

Reply:
Not yet. Depending on feedback to this PR more changes may need to go into Spicy or spicy-plugin. Before the next Zeek release we should have cut releases of Spicy and spicy-plugin though, so I am leaving this open for now.